compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/messages/MessageCollector.kt

```diff
@@ -21,9 +21,23 @@ interface MessageCollector {
 
     fun report(severity: CompilerMessageSeverity, message: String, location: CompilerMessageLocation? = null)
 
+    @Suppress("INAPPLICABLE_JVM_NAME")
```

Comment:
Let's please avoid suppressing errors in the compiler codebase. What are you trying to achieve here?

Reply:
I want to have `JvmName` for interface method which is not allowed without suppression. I'd use the name different from `report` here,but @ligee insists on using this name: https://github.com/JetBrains/kotlin/pull/3443#discussion_r433151327

Reply:
Why should this be an interface method though?

Reply:
Mostly because it should be used in `MessageCollectorBasedReporter#report` which uses instance of MessageCollector, not an instance of a subclass

Reply:
OK but why does it need some other JVM name? Why can't it be just `report`? If you're worried about ambiguities in Java code, you can fix them by casting `null` to an appropriate type:
```
messageCollector.report(..., ..., (CompilerMessageLocation) null);
```

Reply:
Also, if you annotate it with `@JvmDefault`, you will avoid the need to override it in Java subclasses

Reply:
Added JvmDefault and got rid of error suppression